src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index];
+    }
+
+    public enum Color {
```

Comment:
The enum `Color` should be defined in its own file. It's a good practice to keep each top-level class or enum in separate files.